keanu-project/src/test/java/io/improbable/keanu/vertices/dbl/probabilistic/StudentTVertexTest.java

```diff
@@ -0,0 +1,165 @@
+package io.improbable.keanu.vertices.dbl.probabilistic;
+
+import org.apache.commons.math3.distribution.TDistribution;
+import org.apache.commons.math3.stat.descriptive.SummaryStatistics;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static java.lang.Math.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+import static junit.framework.TestCase.assertEquals;
+
+public class StudentTVertexTest {
+	
+	private static final double DELTA = 0.0001;
+	
+	private static int POS_V = 0;
+	private static int POS_MEAN = 1;
+	private static int POS_SD = 2; // standard deviation
+	private static final double[] TEST_VALUES = new double[]{
+			1.0, 2.0, 3.0
+	};
+	
+	private final Logger log = LoggerFactory.getLogger(StudentTVertexTest.class);
+	
+	private Random random;
+	private double mu = 0.;
+	private double sigma = 1.;
+	
+	@Before
+	public void setup() {
+		random = new Random(1);
+	}
+	
+	@Test
+	public void sampleTest() {
+		int N = 10000000;
```

Comment:
That's surprising you need 10million samples to get within 0.1. Is it both the std and mean?

Reply:
Also, does the TDistribution from apache math need this many samples?

Reply:
I don't think the apache one doesn't, ours did. I tried 100k, 1 mil, 10 mil. There was a huge variance with the std deviation.